### PR TITLE
Update fluent-plugin-loki documentation URLs

### PR DESCRIPTION
**Pull request by timothydlister**
Updated fluent-plugin-loki documentation links to point to new location. Original reference docs were moved in this [commit](https://github.com/grafana/loki/commit/6e67d1ea5d128c15e6b4c69895057891c1d1a8f9#diff-04dc5b2136328a0dcb189df97734c7c72e5e1227fa0c03469a6ce608f32f1b66).

**What this PR does / why we need it**:
Update to fluent-plugin-loki documented.
**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [X] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Review by sandeepsukhani — APPROVED** [review]
LGTM. Thanks for the PR!

File: docs/sources/clients/fluentd/_index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ fluent-gem install fluent-plugin-grafana-loki
 
 ## Docker Image
 
-There is a Docker image `grafana/fluent-plugin-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/master/fluentd/fluent-plugin-loki/docker/conf). By default, fluentd containers use the configurations but you can also specify your `fluentd.conf` with `FLUENTD_CONF` environment variable.
+There is a Docker image `grafana/fluent-plugin-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/main/clients/cmd/fluentd/docker/conf). By default, fluentd containers use the configurations but you can also specify your `fluentd.conf` with `FLUENTD_CONF` environment variable.
```
Comment (KMiller-Grafana) [inline docs/sources/clients/fluentd/_index.md:24]: I find this paragraph confusing.  Here's my suggested rewrite, given my questionable understanding of what was meant to be conveyed.

```suggestion
The Docker image `grafana/fluent-plugin-loki:master` contains [default configuration files](https://github.com/grafana/loki/tree/main/clients/cmd/fluentd/docker/conf). By default, fluentd containers use that default configuration. You can instead specify your `fluentd.conf` configuration file with a `FLUENTD_CONF` environment variable.
```

File: docs/sources/clients/fluentd/_index.md [hunk 2]
```diff
@@ -180,7 +180,7 @@ This plugin automatically adds a `fluentd_thread` label with the name of the buf
 ### url
 
 The url of the Loki server to send logs to.  When sending data the publish path (`../api/loki/v1/push`) will automatically be appended.
```
Comment (KMiller-Grafana) [inline docs/sources/clients/fluentd/_index.md:182]: Please also update this prose, even though your PR did not originally touch this prose.

Comment (KMiller-Grafana) [inline docs/sources/clients/fluentd/_index.md:182]: Please also update this prose, even though your PR did not originally touch this prose.

```suggestion
### URL

The URL of the Loki server to send logs to.  When sending data, the publish path (`../api/loki/v1/push`) will automatically be appended.
```

**Review by KMiller-Grafana — APPROVED** [review]
I approve, given the suggested changes.

**Pull request merged**